shadows/httpclient/src/main/java/org/robolectric/shadows/httpclient/ShadowDefaultRequestDirector.java

```diff
@@ -153,6 +155,8 @@ public static HttpRequestInfo getLatestSentHttpRequestInfo() {
   }
 
   /**
+   * Get the {@link HttpRequestInfo} for the given index.
```

Comment:
What about "Get the sent {@link HttpRequestInfo} for the given index"?

Reply:
Sure

Reply:
https://github.com/robolectric/robolectric/pull/7399/files#diff-462c07a5e5d173b71bc1b609db060848db01f654fec2781fe1f5283fd6960575R159